microsoft-edge/devtools-guide-chromium/experimental-features/share-traces.md

```diff
@@ -0,0 +1,177 @@
+---
+title: Share enhanced performance and memory traces
+description: Learn to export and import enhanced performance and memory traces in DevTools to resolve low performance and high memory usage issues in your webpages
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.date: 01/06/2023
+---
+# Share enhanced performance and memory traces
+
+The **Performance** and **Memory** tools record runtime data about your webpage. Exploring the recorded data makes it possible to improve your webpage's memory usage or runtime performance.
+
+Additionally, the recorded data can be exported to files on disk. The exported files are called _traces_.
+
+Exporting performance and memory traces is useful when you want to share these files with other people to get help with investigating issues. An exported trace is a `.json`, `.heapsnapshot`, `.heapprofile`, or `.heaptimeline` file which can be imported in DevTools at any time. By default, traces contain minimal information about the runtime data from the traced webpage.
+
+Starting with Microsoft Edge 109, traces can be exported as _ enhanced traces_ as an experimental feature.
```

Comment:
```suggestion
Starting with Microsoft Edge 109, traces can be exported as _enhanced traces_ as an experimental feature.
```

Reply:
Recomm. comma:
Starting with Microsoft Edge 109, traces can be exported as _enhanced traces_, as an experimental feature.